beest/sut/bees/tasks/install-and-update/install-stackstate-agent.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Why do we need these ansible files? Remove

Reply:
agreed